src/app/features/home/capture-tab/capture-details/capture-details.page.ts

```diff
@@ -166,6 +166,26 @@ export class CaptureDetailsPage {
       .subscribe();
   }
 
+  openMap() {
+    return this.proof$
+      .pipe(
+        concatMap(proof =>
+          iif(
+            () =>
+              proof.geolocationLatitude !== undefined &&
+              proof.geolocationLongitude !== undefined,
+            defer(() =>
+              Browser.open({
+                url: `https://maps.google.com/maps?q=${proof.geolocationLatitude}, ${proof.geolocationLongitude}`,
```

Comment:
```suggestion
                url: `https://maps.google.com/maps?q=${proof.geolocationLatitude},${proof.geolocationLongitude}`,
```

Drop the space between latitude and longitude, it's not necessary and it prevents the iOS App from opening the url successfully. (It seems the iOS browser won't do the url encoding automatically. Instead, it reports "Invalid URL" and refuses to open the link.)